src/Microsoft.SqlTools.ServiceLayer/MachineLearningServices/Contracts/ExternalLanguageStatusRequest.cs

```diff
@@ -0,0 +1,43 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using Microsoft.SqlTools.Hosting.Protocol.Contracts;
+
+namespace Microsoft.SqlTools.ServiceLayer.MachineLearningServices.Contracts
+{
+    public class ExternalLanguageStatusRequestParams
+    {
+        /// <summary>
+        /// Connection uri
+        /// </summary>
+        public string OwnerUri { get; set; }
+
+        /// <summary>
+        /// Language name
+        /// </summary>
+        public string LanguageName { get; set; }
+    }
+
+    /// <summary>
+    /// Response class for external language status
+    /// </summary>
+    public class ExternalLanguageStatusResponseParams
+    {
+        /// <summary>
+        /// Language status
+        /// </summary>
+        public bool Status { get; set; }
+    }
+
+    /// <summary>
+    /// Request class for external language status
+    /// </summary>
+    public class ExternalLanguageStatusRequest
+    {
+        public static readonly
+            RequestType<ExternalLanguageStatusRequestParams, ExternalLanguageStatusResponseParams> Type =
+                RequestType<ExternalLanguageStatusRequestParams, ExternalLanguageStatusResponseParams>.Create("mls/externallanguagestatus");
```

Comment:
camel case here and for other request types

Reply:
I remember we use lower case for all request types. I'll double check

---
In reply to: [364373242](https://github.com/microsoft/sqltoolsservice/pull/906#discussion_r364373242) [](ancestors = 364373242)